_i18n/ar/resources/developer-guides/daemon-rpc.md

```diff
@@ -0,0 +1,2423 @@
+## Introduction
```

Comment:
add the "untranslated snippet"(`{% include untranslated.html %}`) before `## introduction` so that the page will look like:

```
{% include untranslated.html %}

## Introduction
+
+This is a list of the monerod daemon RPC calls, their inputs and outputs, and examples of each.
+
+Many RPC calls use the daemon's JSON RPC interface while others use their own interfaces, as demonstrated below.
+
...
```